addons/kommander/1.0.x/kommander.yaml

```diff
@@ -41,25 +41,14 @@ spec:
   chartReference:
     chart: kommander
     repo: https://mesosphere.github.io/charts/stable
-    version: 0.4.26
+    version: 0.4.30
     values: |
       ---
       ingress:
         extraAnnotations:
           traefik.ingress.kubernetes.io/priority: "2"
 
       kommander-cluster-lifecycle:
-        controller:
-          containers:
-            manager:
-              image:
-                tag: v0.4.12
```

Comment:
Is this right? The chart values have no tag. Are we expecting to use `latest`?

from _kommander-cluster-lifecycle-v0.5.7.tgz_ `values.yaml`
```
controller:
  containers:
    manager:
      extraArgs: {}
      replicas: 1
      image:
        repository: mesosphere/kommander-cluster-lifecycle-controller-manager
        tag:
...
webhook:
  extraArgs: {}
  image:
    repository: mesosphere/kommander-cluster-lifecycle-webhook
    tag:
...
utilityApiserver:
  image:
    repository: mesosphere/kommander-cluster-lifecycle-utility-apiserver
    tag:
```

Reply:
Yes the default is the chart version (see e.g. https://github.com/mesosphere/kommander-cluster-lifecycle/blob/master/chart/kommander-cluster-lifecycle/templates/deployment_controllermanager.yaml#L31). These overrides were added to try to use the specific image version without having to update kommander chart, but this didn't work (CRDs weren't updated and needed the kommander chart update to work).

Reply:
Ah, cool. Thanks.